pages/vote.tsx

```diff
@@ -216,9 +186,9 @@ class VotePage extends React.Component<VoteProps, VoteState> {
             <div className="col-md-8" style={{ backgroundColor: '#f5f5f5', padding: '0 20px' }}>
               <h2 style={{ marginTop: '30px' }}>Getting the most out of voting</h2>
               <p>
-                This year we had {this.state.sessions ? this.state.sessions.length : '...'} sessions submitted! We've
-                implemented the following features to assist you to manage voting across such a large number of
-                sessions:
+                This year we had {!this.state.isLoading && this.state.sessions ? this.state.sessions.length : '...'}{' '}
```

Comment:
Should we do something for `isError`?

Reply:
Fair. In the scenario of an error we would currently be displaying zero sessions.